server/app/views/applicant/ApplicantInformationView.java

```diff
@@ -93,7 +117,65 @@ public Content render(
             .addMainStyles(ApplicantStyles.MAIN_APPLICANT_INFO)
             .addMainContent(h1(title).withClasses("sr-only"), formContent);
```

Comment:
Below, we add all the dev information that we previously had on the login page to the language selection page. See the PR description for a screenshot.